apps/api/src/domain/models/media-item/commands/media-item-published.event.ts

```diff
@@ -0,0 +1,22 @@
+import { AggregateId } from '../../../types/AggregateId';
+import BaseDomainModel from '../../BaseDomainModel';
+import { IEvent } from '../../shared/events/interfaces/event.interface';
+import { EventRecordMetadata } from '../../song/commands/song-created.event';
+import { PublishMediaItem } from './publish-media-item.command';
+
+export class MediaItemPublished extends BaseDomainModel implements IEvent {
+    type = 'MEDIA_ITEM_PUBLISHED';
+
+    meta: EventRecordMetadata;
+
+    constructor(command: PublishMediaItem, eventId: AggregateId) {
+        super();
+
+        Object.assign(this, command);
```

Comment:
With the next one of these, I'd consider finding a way to reuse this logic. Maybe make `IEvent` into an `EventBase` instead.